draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1122,10 +1122,6 @@ Outputs:
     `tree_size_1` MUST match the `first` input. If the `sth` output is omitted,
     then `tree_size_2` MUST match the `second` input.
 
-  sth:
-  : A base64 encoded `TransItem` of type `signed_tree_head_v2`, signed by this
-    log.
-
```

Comment:
Please also update the paragraph just above the "Outputs:", which still refers to this deleted "sth" output parameter.  It currently says:
```Both tree sizes must be from existing v2 STHs.  However, because of skew, the receiving front-end may not know one or both of the existing STHs.  If both are known, then only the "consistency" output is returned.  If the first is known but the second is not (or has been omitted), then the latest known STH is returned, along with a consistency proof between the first STH and the latest.  If neither are known, then the latest known STH is returned without a consistency proof.```